.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,46 @@
+<!--
+For high severity issues, please contact AWS Support directly.
+
+Please provide the following information:
+-->
+
+
+### Summary
+<!-- Please provide a brief outline of the issue -->
+
+
+### Description
+<!-- Provide detailed information about this issue -->
+
+
+
+<!--
+Please use these blocks to comment out appropriate sections
+especially for feature requests
+-->
+### Expected Behavior
+
+
+### Environment Details
+<!--
+Examples:
+* docker info
+* curl http://localhost:51678/v1/metadata
+* df -kh
+-->
+
+
+### Supporting Log Snippets
```

Comment:
On that note, `please remove sensitive data` is a very generic message. As a customer, it's very hard to parse Agent and other logs to determine if they contain any sensitive data. Should we provide more details here? Or should the guidance be that logs should be collected, but we'll provide a mechanism to share them with ECS team after the issue has been created?